src/collections/audioTactileCards/audioTactileCards.11tydata.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
```

Comment:
This can be restructured to match the [directory data files](https://github.com/fluid-project/trivet/blob/main/src/collections/pages/pages.11tydata.js#L3) in Trivet. This will also necessate adding [`en-CA.11tydata.js`](https://github.com/fluid-project/trivet/blob/main/src/collections/pages/en-CA/en-CA.11tydata.js) files in the specific locale directory.